Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py

```diff
@@ -400,6 +557,124 @@ def get_scan_status_command():
     return get_entry_for_object('Scan status for {}'.format(scan_id), 'TenableIO.Scan(val.Id === obj.Id)', scan_status)
 
 
+def pause_scan_command():
+    """Function for integration command pause_scan_command."""
+    scan_ids = argToList(demisto.getArg('scanIds'))
+
+    results = []
+
+    for scan_id in scan_ids:
+
+        scan_details = send_scan_request(scan_id)
+        scan_status = {
+            'Id': scan_id,
+            'Status': scan_details.get('info').get('status')
+        }
+
+        if scan_status["Status"].lower() == "running":
+            send_scan_request(scan_id, "pause", "POST")
+            resumed_scan = {
+                "Id": scan_id,
+                "Status": "Pausing"
+            }
+            results.append(get_entry_for_object("The requested scan was paused successfully",
+                                                "TenableIO.Scan", replace_keys(resumed_scan), ["Id", "Status"]))
+
+        else:
+            results.append(
+                "Command 'tenable-io-pause-scan' cannot be "
+                "called while scan status is {} for scanID {}".format(scan_status["Status"], scan_id))
+
+    return results
+
+
+def resume_scan_command():
+    """Function for integration command resume_scan_command."""
+    scan_ids = argToList(demisto.getArg('scanIds'))
+
+    results = []
+
+    for scan_id in scan_ids:
+
+        scan_details = send_scan_request(scan_id)
+        scan_status = {
+            'Id': scan_id,
+            'Status': scan_details.get('info').get('status')
+        }
+
+        if scan_status["Status"].lower() == "paused":
+            send_scan_request(scan_id, "resume", "POST")
+            resumed_scan = {
+                "Id": scan_id,
+                "Status": "Resuming"
+            }
+            results.append(get_entry_for_object("The requested scan was resumed successfully",
+                                                "TenableIO.Scan", replace_keys(resumed_scan), ["Id", "Status"]))
+
+        else:
+            results.append(
+                "Command 'tenable-io-resume-scans' cannot be "
+                "called while scan status is {} for scanID {}".format(scan_status["Status"], scan_id))
+
+    return results
+
+
+def get_scan_templates():
+    """Function for integration command get_scan_templates."""
+    try:
+        endpoint = BASE_URL + "editor/scan/templates"
+        response = requests.request("GET", endpoint, headers=NEW_HEADERS, verify=USE_SSL)
+        response.raise_for_status()
+        #print(response.json())
+        demisto.info("Ran request sucessfully")
+        return response
+    except HTTPError:
+        #print(response.status_code)
+        demisto.error(traceback.format_exc())
+        sys.exit(0)
+
+
+# Request/Response methods
+# kwargs: request parameters
```

Comment:
Let's remove these comments

Reply:
Hi @guykeller (cc: @barchen1 ), I am committing a new version of Tenable_io.py where I removed / cleaned up the commenting.

Reply:
Done
Update 02.23.2021 1034 HRS EST:
Removed comments from code